### PR TITLE
Add test steps into README

### PR DESCRIPTION
**Pull request by yuxisun1217**
Signed-off-by: Yuxin Sun <yuxisun@redhat.com>

### CODE REVIEW

**Review by teg — COMMENTED** [review]
Thanks for this @yuxisun1217! This looks great! I'll let @Gundersanne have a look before merging, but looks good to me!

File: README.md [hunk 1]
```diff
@@ -3,5 +3,68 @@
 
 ## Testing
 
-Coverage report is available from
+### Preparation
+1. Clone the following repositories:
+    * https://github.com/osbuild/image-builder-frontend
```
Comment (Gundersanne) [inline README.md:8]: I think we can leave the image-builder-frontend and insights-proxy setup out of the image-builder readme.

File: README.md [hunk 2]
```diff
@@ -3,5 +3,68 @@
...
+    * https://github.com/osbuild/image-builder
+    * https://github.com/RedHatInsights/insights-proxy
+
+2. Setting up the proxy
```
Comment (Gundersanne) [inline README.md:12]: See above, the proxy is not required to run the integration tests.

File: README.md [hunk 3]
```diff
@@ -3,5 +3,68 @@
...
+     -e OSBUILD_CA_PATH=/etc/osbuild-composer/ca-crt.pem \
+     -e OSBUILD_CERT_PATH=/etc/osbuild-composer/client-crt.pem \
+     -e OSBUILD_KEY_PATH=/etc/osbuild-composer/client-key.pem \
+     -v /etc/osbuild-composer:/etc/osbuild-composer \
+     image-builder # Start image-builder container
+    # go test ./cmd/... -tags=integration # Run integration test
+    ```
+   
+### CI
+* https://osbuildci.cloud.paas.psi.redhat.com/me/my-views/view/all/job/osbuild/job/image-builder/job/main/
```
Comment (Gundersanne) [inline README.md:65]: This is an internal url, personally don't think it should be in the public readme.

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Thanks! Looks good, some minor change requests.

**Comment by yuxisun1217** [comment]
Thank @Gundersanne ! I removed the image-builder-frontend and insights-proxy repos, removed CI section, and cut some redundant words. Please help to review again.

**Comment by yuxisun1217** [comment]
BTW, what's our custom here if I want to change code in a PR? Force push to the same commit or create a new commit into the same branch? I used to force push before. Thanks!

**Review by Gundersanne — APPROVED** [review]
Nice, thank you

Force pushing is best, alternatively adding commits on top and then using github to squash is also fine (though that only works when there needs to be only 1 commit merged in the end).

**Pull request merged**